netlify.toml

```diff
@@ -0,0 +1,46 @@
+########################################
+# Bonita redirects
+########################################
+[[redirects]]
+  from = "/bonita/latest/*"
+# TODO if we want proxy/rewrite instead, add status = 200 (https://docs.netlify.com/routing/redirects/rewrites-proxies/)
+  to = "/bonita/2021.1/:splat"
+
+[[redirects]]
+  from = "/bonita/"
+  to = "/bonita/2021.1/"
+
+# Special redirect after introduction of Bonita branding version
+# TODO we may prefer to have 2021.1 in the doc and use a dedicated redirect from 7.12 to 2021.1 instead
```

Comment:
comment not accurate anymore as @benjaminParisel has done the job to directly publish 2021.1 instead of 7.12
```suggestion
```